### ISSUE TITLE
宽高加载出现与控件不一致

### ISSUE THREAD

**Issue by zhukai123** [penfeizhou/APNG4Android#177]
### New Issue Checklist

- [x] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | e.g. Huawei Mate 30 pro                               |      |
| System Version               | e.g.  6.0                                             |      |
| APNG4Android Library Version | e.g.  1.0.0                                           |      |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo project  | e.g. does it happen with our demo project?            |      |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by zhukai123** [comment]
我的需求是宽高为控件大小, 但偶尔出现Apng在ImageView展示为原图的宽高

**Comment by penfeizhou** [comment]
有可以复现的demo代码吗
